packages/interface-internal/src/registrar.ts

```diff
@@ -1,5 +1,10 @@
 import type { StreamHandler, StreamHandlerOptions, StreamHandlerRecord, Topology, IncomingStreamData } from '@libp2p/interface'
 
+/**
```

Comment:
I'm not sure this comment is necessary - these are named exports so will get their own documentation blocks.

Please can you check the output of `npm run docs` (look in the `.docs` folder) and delete this comment if it doesn't appear anywhere.